smalltalksrc/VMMakerTests/VMSimpleStackBasedCogitAbstractTest.class.st

```diff
@@ -466,11 +460,14 @@ VMSimpleStackBasedCogitAbstractTest >> executePrimitiveWithReceiver: receiverOop
 	  - the receiver goes to the receiver register
 	  - the argument should be pushed to the stack
 	If we are in a system without a link register, we need to push the caller IP to the stack to simulate a call"
+
 	self
 		prepareStackForPrimitiveReceiver: (self twoComplementFor: receiverOop)
 		arguments: (aCollection collect: [ :arg | self twoComplementFor: arg ])
 		method: aMethod.
 
+"self openMachineDebugger . 1halt."
```

Comment:
Is it necessary to commit these comments? :)
Otherwise the PR looks fine

Reply:
Yes, super important 👀 
```suggestion
```